### PR TITLE
Add helper neurom.check.ok

### PR DESCRIPTION
**Pull request by juanchopanza**
Returns sensible values of True or False for morphology checks:

True / False ==> True / False
Iterable ==> True if empty else False

### CODE REVIEW

**Comment by juanchopanza** [comment]
@jdcourcol This should simplify the checks.

**Comment by jdcourcol** [comment]
agree. thank you.

**Comment by juanchopanza** [comment]
@eleftherioszisis @lidakanari FYI some more simplifications to morphology checking.

File: .travis.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ before_install:
 install:
   - pip install --upgrade pip
   - pip install --ignore-installed -r requirements_dev.txt
-  - pip install matplotlib==1.3.1
+  - pip install matplotlib==1.4.1
```
Comment (lidakanari) [inline .travis.yml]: Out of curiosity, why do we modify the matplotlib version?

**Comment by lidakanari** [comment]
I like the structure with the tests for this :+1:

File: .travis.yml (hunk 1 above)
Comment (juanchopanza) [inline .travis.yml]: Travis moved their infrastructure, and the matplotlib build started failing. There was an issue with it not working with the installed numpy, which had been fixed in newer versions. I don't know how or why it worked before.

Comment (lidakanari) [inline .travis.yml]: Ok, we need to check if all the plotting functionality is compatible with this version, but if no tests are failing that's a good start. Good to know.

**Pull request merged**